src/models/student.js

```diff
@@ -0,0 +1,54 @@
+const mongoose = require('mongoose');
+
+const subject = mongoose.Schema({
+  title: {
+    type: String,
+    enum: ['Math', 'Physics', 'Biology', 'History', 'Programming'],
+    default: 'Math',
+    required: true,
+  },
+});
+
+const student = mongoose.schema({
+  username: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  firstName: {
+    type: String,
+    required: true,
+  },
+  lastName: {
+    type: String,
+    required: true,
+  },
+  password_hash: {
+    type: String,
+    required: true,
+  },
+  email: {
+    type: String,
+    required: true,
+  },
+  isTutor: {
+    type: Boolean,
+    default: false,
+  },
+  avatar: {
+    type: String,
+  },
+  subjects: {
+    type: [subject],
+  },
+  questions: {
+    type: [mongoose.Schema.Types.ObjectId],
+    ref: 'Question',
+  },
+  inbox: {
```

Comment:
I would give this the same name of the referenced schema which is `messagingChannel` so it's not confusing